### PR TITLE
Add patched version of crypropp project

### PR DESCRIPTION
**Pull request by dmakarov**

### CODE REVIEW

File: cmake/projects/cryptopp/hunter.cmake [hunk 1]
```diff
@@ -8,6 +8,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 
 # List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    cryptopp
+    VERSION
+    "5.6.5-p0"
```
Comment (ruslo) [inline cmake/projects/cryptopp/hunter.cmake]: Update default version in `cmake/configs/default.cmake`:
* https://docs.hunter.sh/en/latest/creating-new/update.html#update-package

Comment (dmakarov) [reply cmake/projects/cryptopp/hunter.cmake]: @ruslo: updated.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2880
* https://travis-ci.org/ingenue/hunter/builds/307846314?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2881
* https://travis-ci.org/ingenue/hunter/builds/307897959?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.173

**Pull request merged**